### DISCUSSION TITLE
Add Message To All Friends Plugin

### DISCUSSION THREAD

**Discussion by yowaddup100** [runelite/runelite#16939] [Ideas]
Would be cool to add a message to all friend's feature. Would be nice if looking for people to pvm with and you want to prioritize playing with people who earn the right to be your osrs friend. A lot of high-level clan members prefer to play with their friends > new clan mates. 

Even for lower-level players who are adding a lot of friends, it would be nice to stay in touch with all of them at the same time.

**Comment by Nightfirecat** [comment]
We cannot implement this, that would be macroing.
